components/inputs/checkbox.qmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "Checkbox"
+sidebar: components 
+previewapp: |
+  from shiny import App, render, ui
+  app_ui = ui.page_fluid(
+      ui.input_checkbox("x", "Checkbox Checkbox").add_class("mb-0"),
+      ui.output_text_verbatim("txt"),
+      {"class": "vh-100 d-flex justify-content-center align-items-center px-4"}
+  )
+  def server(input, output, session):
+      @output
+      @render.text
+      def txt():
+          return f""
+  app = App(app_ui, server, debug=True)
```

Comment:
The `debug=True` causes messages to be printed to the JS console, but I don't think those are helpful for this page. So I'd suggest removing all of them.

```suggestion
  app = App(app_ui, server)
```